src/Trixi.jl

```diff
@@ -24,26 +24,27 @@ using Printf: @printf, @sprintf, println
 # import @reexport now to make it available for further imports/exports
 using Reexport: @reexport
 
-import DiffEqBase: @muladd, CallbackSet, DiscreteCallback,
-                   ODEProblem, ODESolution, ODEFunction,
-                   get_du, get_tmp_cache, u_modified!,
-                   get_proposed_dt, set_proposed_dt!, terminate!, remake
+using DiffEqBase: @muladd, CallbackSet, DiscreteCallback,
+                  ODEProblem, ODESolution, ODEFunction
+import DiffEqBase: get_du, get_tmp_cache, u_modified!,
+                   get_proposed_dt, set_proposed_dt!, 
+                   terminate!, remake
 using CodeTracking: code_string
 @reexport using EllipsisNotation # ..
-import ForwardDiff
+using ForwardDiff: ForwardDiff
 using HDF5: h5open, attributes
 using LinearMaps: LinearMap
 using LoopVectorization: LoopVectorization, @turbo, indices
 using LoopVectorization.ArrayInterface: static_length
-import MPI
+using MPI: MPI
 using GeometryBasics: GeometryBasics
 using Octavian: matmul!
 using Polyester: @batch # You know, the cheapest threads you can find...
 using OffsetArrays: OffsetArray, OffsetVector
 using P4est
-using RecipesBase
-using Requires
-using SparseArrays: sparse, droptol!, rowvals, nzrange, AbstractSparseMatrix
+using RecipesBase: RecipesBase, @recipe, @series
```

Comment:
I think we didn't include at-recipe because it conflicts with Makie's at-recipe. This is also why we always use the qualified RecipesBase.at-recipe. But I might be mistaken, thus CC'ing @jlchan here.

Reply:
Makie's `@recipe` is always prefixed - I added the prefix `RecipesBase` to all macros from that package and removed the corresponding `using` statements in https://github.com/trixi-framework/Trixi.jl/pull/839/commits/2b7b95e2a7e22e4059d131723174dec7b9ebffc0

Reply:
👍